test/transactionBreakdown.test.ts

```diff
@@ -0,0 +1,92 @@
+import { connectDatabase, disconnectDatabase } from '../utils/test-utils/db.util';
+import supertest from 'supertest';
+import testApp from '../app';
+import _ from 'lodash';
+const request = supertest(testApp);
+
+import transactionBreakdownSeed from './transactionBreakdown.test.data';
+
+describe('transactionBreakdown routes', () => {
+  beforeAll(() => {
+    connectDatabase();
+  });
+
+  afterAll(() => {
+    disconnectDatabase();
+  });
+
+  describe('POST route', () => {
+    it('should return a transactionBreakdown with accurate orderTransaction', async () => {
+      const res = await request
+        .post('/api/transactionBreakdown')
+        .send(transactionBreakdownSeed.transactionBreakdownNormal);
+      const { orderTransaction } = res.body;
+
+      expect(res.status).toBe(200);
+      expect(
+        _.isEqual(
+          orderTransaction,
+          transactionBreakdownSeed.transactionNormalComparison.orderTransaction
+        )
+      ).toBe(true);
+    });
+
+    it('should return a transactionBreakdown with accurate orderTransactionPersons', async () => {
```

Comment:
Can't this be combined with L19 unit test? We're testing the route with the same input.